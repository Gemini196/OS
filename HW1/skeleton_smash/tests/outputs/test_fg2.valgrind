==41029== Memcheck, a memory error detector
==41029== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==41029== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==41029== Command: /home/student/Desktop/OS/HW1/skeleton_smash/smash
==41029== Parent PID: 41025
==41029== 
==41029== 
==41029== FILE DESCRIPTORS: 6 open at exit.
==41029== Open file descriptor 1: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_fg2.out
==41029==    at 0x54EF957: dup2 (syscall-template.S:78)
==41029==    by 0x1109FD: main (smash.cpp:41)
==41029== 
==41029== Open file descriptor 0:
==41029==    at 0x54EF957: dup2 (syscall-template.S:78)
==41029==    by 0x1109AB: main (smash.cpp:33)
==41029== 
==41029== Open file descriptor 5:
==41029==    at 0x54EF927: dup (syscall-template.S:78)
==41029==    by 0x110B4B: __static_initialization_and_destruction_0(int, int) (smash.cpp:9)
==41029==    by 0x110B67: _GLOBAL__sub_I_stdout_dp (smash.cpp:57)
==41029==    by 0x110BEC: __libc_csu_init (in /home/student/Desktop/OS/HW1/skeleton_smash/smash)
==41029==    by 0x5400C17: (below main) (libc-start.c:266)
==41029== 
==41029== Open file descriptor 4: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_fg2.out
==41029==    at 0x54EF927: dup (syscall-template.S:78)
==41029==    by 0x110B3B: __static_initialization_and_destruction_0(int, int) (smash.cpp:8)
==41029==    by 0x110B67: _GLOBAL__sub_I_stdout_dp (smash.cpp:57)
==41029==    by 0x110BEC: __libc_csu_init (in /home/student/Desktop/OS/HW1/skeleton_smash/smash)
==41029==    by 0x5400C17: (below main) (libc-start.c:266)
==41029== 
==41029== Open file descriptor 3: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_fg2.valgrind
==41029==    <inherited from parent>
==41029== 
==41029== Open file descriptor 2: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_fg2.err
==41029==    <inherited from parent>
==41029== 
==41029== 
==41029== HEAP SUMMARY:
==41029==     in use at exit: 0 bytes in 0 blocks
==41029==   total heap usage: 55 allocs, 55 frees, 84,993 bytes allocated
==41029== 
==41029== All heap blocks were freed -- no leaks are possible
==41029== 
==41029== For counts of detected and suppressed errors, rerun with: -v
==41029== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
