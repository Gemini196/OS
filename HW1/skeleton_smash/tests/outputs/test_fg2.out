smash> smash> smash> smash> smash> smash> sleep 300& : 46955
smash: got ctrl-C
smash: process 46955 was killed
smash> sleep 100& : 46953
smash: got ctrl-C
smash: process 46953 was killed
smash> sleep 200& : 46954
smash: got ctrl-C
smash: process 46954 was killed
smash> sleep 400& : 46956
smash: got ctrl-C
smash: process 46956 was killed
smash> sleep 500& : 46957
smash: got ctrl-C
smash: process 46957 was killed
smash> smash: sending SIGKILL signal to 0 jobs:
